.eslintrc

```diff
@@ -0,0 +1,258 @@
+{
```

Comment:
https://eslint.org/

there are editor integrations, if you use vscode or atom or sublime or whatevs

Reply:
Oh nice! cool.